baseline/OSPS-BR.yaml

```diff
@@ -221,15 +221,18 @@ controls:
       - id: OSPS-BR-03.02
         text: |
           When the project lists a URI as an official distribution channel,
-          that URI MUST be exclusively delivered using encrypted channels.
+          that channel MUST be protected from adversary-in-the-middle
+          attacks.
```

Comment:
1. What is the functional difference between the old and new text?
2. If there is value added by this, why isn't `BR-03.01` also changed to get the same value?